draft-ietf-teep-protocol.md

```diff
@@ -221,7 +221,7 @@ otherwise.
 
 # Detailed Messages Specification {#detailmsg}
 
-TEEP messages are protected by the COSE_Sign1 structure.
+TEEP messages are protected by the COSE_Sign1 or COSE_Sign structure. The initial QueryRequest message MAY have COSE_Sign structure with both ES256 (cose-alg-es256) and EdDSA (cose-alg-eddsa) as described in {{ciphersuite}}. 
```

Comment:
```suggestion
TEEP messages are protected by the COSE_Sign1 or COSE_Sign structure as described in {{ciphersuite}}. 
```
Don't introduce a MAY here, leave it for {{ciphersuite}} which already does.

Reply:
@dthaler Thanks for your review. Removed descriptions which are in {{ciphersuite}} already.